web/packages/teleterm/src/main.ts

```diff
@@ -194,6 +195,29 @@ async function initializeApp(): Promise<void> {
   });
 }
 
+/**
+ * There is an outstanding issue about Electron storing its caches in the wrong location https://github.com/electron/electron/issues/8124.
+ * Based on the Apple Developer docs and the discussion under that issue, changing the location of `sessionData`
+ * to `~/Library/Caches` on macOS and `XDG_CACHE_HOME` or `~/.cache`
+ * (https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) on Linux seems like a reasonable thing to do.
+ */
+function updateSessionDataPath() {
```

Comment:
Do you think we could update it for Windows too while we're at it?

* https://github.com/electron/electron/issues/8124#issuecomment-277490980
* https://github.com/electron/electron/issues/1404

I think it'd be something like

```typescript
path.resolve(process.env.LOCALAPPDATA, app.getName())
```

But I'm not 100% sure, worth checking if `LOCALAPPDATA` is always in env (it most probably is, but that's just my assumption).

https://stackoverflow.com/questions/19275776/node-js-how-to-get-the-os-platforms-user-data-folder

Reply:
I tested that in node REPL and the variable was set, but I will test it in a packaged app as well, to be 100% sure.